templates/kubernetes/docs/install-offline.md

```diff
@@ -262,8 +262,8 @@ is covered in the [proxy documentation][].
 [juju-bundle]: https://juju.is/docs/sdk/bundles
 [juju-constraints]: https://juju.is/docs/olm/constraints
 [juju-docs]: https://juju.is/docs/olm/installing-juju
-[juju-gui]: https://juju.is/docs/olm/accessing-juju%E2%80%99s-web-interface
-[offline-mode]: https://juju.is/docs/t/offline-mode-strategies/1071
+[juju-gui]: https://juju.is/docs/olm/accessing-the-dashboard
```

Comment:
I'm not sure the `[juju-gui]` link is used in this doc - I searched the codebase, and there's another instance of it on [/kubernetes/docs/install-manual](https://github.com/canonical-web-and-design/ubuntu.com/blob/main/templates/kubernetes/docs/install-manual.md), but it's using the `https://juju.is/docs/olm/accessing-juju%E2%80%99s-web-interface` URL

Reply:
a lot of the juju docs continue to move around, which isn't great. But if it isn't used here I will just remove that one